build.gradle.kts

```
@@ -48,7 +48,9 @@ nexusPublishing {
 }
 
 // start - define tasks to download, unzip, and generate from opentelemetry/semantic-conventions
-var generatorVersion = "0.23.0"
+// TODO: build locally from https://github.com/open-telemetry/build-tools/tree/feature/codegen-by-namespace branch
+// Blocked on release of this: https://github.com/open-telemetry/build-tools/pull/243#issuecomment-1936446767
+var generatorVersion = "codegen-by-namespace"
```

```suggestion
var generatorVersion = "feature-codegen-by-namespace"
```
see https://hub.docker.com/r/otel/semconvgen/tags

---

🎉